python/pyspark/sql/types.py

```diff
@@ -1629,35 +1629,121 @@ def to_arrow_type(dt):
     return arrow_type
 
 
-def _check_dataframe_localize_timestamps(pdf):
+def to_arrow_schema(schema):
```

Comment:
where do we use this method?

Reply:
Ah, currently it isn't used. I'll remove it for now.